### ISSUE TITLE
How to define the value of a TextArea widget?

### ISSUE THREAD

**Issue by krillefear** [erlage/rad#11]
In my app I would like to give the output of a calculation as a longer Text in a readonly TextArea, so that it is easy to select and copy the content. But I find no way to define a value in the TextArea constructor:

```dart
TextArea(
            placeholder: 'Dart output',
            readOnly: true,
            maxLength: 9999,
            classAttribute: 'w-full h-96 border mt-4 p-2',
            innerText: outputVariable,
          )
```

I can only define the innerText but this is not the value. I can access the TextArea with `document.getElementById` and then change the value property but after a setState this is lost.

**Comment by erlage** [comment]
Hey @krillefear, 

I see currently it's very difficult to properly set value of a TextArea widget. 

If you're doing direct DOM manipulation, value will presist between setState calls. But it's definitely not an ideal solution as there are number of cases where you won't be able to access dom node so it might not work as expected.

Better way would be to provide a `value` property to `TextArea` widget just like we've in `Input` widgets, and let the framework take care of setting/updating value.

**Comment by krillefear** [comment]
Thanks for the response :) Maybe something like a TextEditingController could be useful in the future as well. Currently I try to just get the DOM object to read/write the value and name the reference to it "textEditingController" which works

**Comment by erlage** [comment]
I really like the idea of having TextEditingControllers. I just don't want to mix Flutter widgets with HTML widgets. What we can do is have a TextField widget(like in Flutter) that can accept text editing controllers. Originally plan was indeed to provide Flutter widgets(MaterialApp, Container, Center etc) with fully compaitable API so that we can render a Flutter app directly in Rad. I still want to create such a package. I just have to finish some foundational work for that package and then you can help implement whatever you want from Flutter:)

Talking about current issue, yesterday I checked React, and they [too provide `value` attribute](https://reactjs.org/docs/dom-elements.html#value) on TextArea so this is something I think we should go for now.

**Comment by erlage** [comment]
TextArea widget now have `value` property that can be used to set/update/unset its value.

Available in v0.10.0
